aten/src/THCUNN/generic/THCUNN.h

```diff
@@ -4,6 +4,66 @@
 
 #include <ATen/core/Reduction.h>
 
+THC_API void THNN_(SpatialAveragePooling_updateOutput)(
+                  THCState *state,
+                  THCTensor *input,
+                  THCTensor *output,
+                  int kW, int kH,
+                  int dW, int dH,
+                  int padW, int padH,
+                  bool ceil_mode,
+                  bool count_include_pad);
+
+THC_API void THNN_(SpatialAveragePooling_updateGradInput)(
```

Comment:
I don't think these backward definitions (updateGradInput, accGradParameters) make sense for integer types.

Reply:
Fixed: https://github.com/pytorch/pytorch/pull/20370